doc/man3/EVP_PKEY_get_default_digest_nid.pod

```diff
@@ -18,8 +18,8 @@ EVP_PKEY_get_default_digest_nid, EVP_PKEY_get_default_digest_name
 EVP_PKEY_get_default_digest_name() fills in the default message digest
 name for the public key signature operations associated with key
 I<pkey> into I<mdname>, up to at most I<mdname_sz> bytes including the
-ending NUL byte.  The name could be C<"UNDEF">, signifying that no digest
-should be used.
+ending NUL byte.  The name could be C<"UNDEF">, signifying that a digest
+must (for return value 2) or may (for return value 1) be left unspecified.
```

Comment:
In this context, the "optional character" of the (mandatory digest) parameter becomes clear (for ret val 1). But then again, there may be people only reading the code and not the documentation ... :-)